gradio/components/multimodal_textbox.py

```diff
@@ -0,0 +1,139 @@
+"""gr.MultimodalTextbox() component."""
+
+from __future__ import annotations
+
+from typing import Any, Callable, Literal
+
+from gradio_client.documentation import document
+
+from gradio.components.base import FormComponent
+from gradio.events import Events
+
+
+@document()
+class MultimodalTextbox(FormComponent):
+    """
+    Creates a textarea for users to enter string input or display string output and also allows for the uploading of multimedia files.
+
+    Demos: chatbot_multimodal
+    Guides: creating-a-chatbot
+    """
+
+    EVENTS = [
+        Events.change,
+        Events.input,
+        Events.select,
+        Events.submit,
+        Events.focus,
+        Events.blur,
+    ]
+
+    def __init__(
+        self,
+        value: list[dict[str, str]] | Callable | None = None,
+        *,
+        file_types: list[str] | None = None,
+        lines: int = 1,
+        max_lines: int = 20,
+        placeholder: str | None = None,
+        label: str | None = None,
+        info: str | None = None,
+        every: float | None = None,
+        show_label: bool | None = None,
+        container: bool = True,
+        scale: int | None = None,
+        min_width: int = 160,
+        interactive: bool | None = None,
+        visible: bool = True,
+        elem_id: str | None = None,
+        autofocus: bool = False,
+        autoscroll: bool = True,
+        elem_classes: list[str] | str | None = None,
+        render: bool = True,
+        text_align: Literal["left", "right"] | None = None,
+        rtl: bool = False,
+        show_copy_button: bool = False,
```

Comment:
I think we should add a parameter for whether or not to show the submit button. I can imagine some non-chatbot use cases where it doesn't make sense